src/util/wren2c.c

```diff
@@ -0,0 +1,81 @@
+
+internal 
+void WREN2C_randomString(char *dest) {
+    // https://stackoverflow.com/a/15768317
+    char charset[] = "0123456789"
+                     "abcdefghijklmnopqrstuvwxyz"
+                     "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+    srand(time(0));
+
+    size_t length = 10;
+    while (length-- > 0) {
+        size_t index = (double) rand() / RAND_MAX * (sizeof charset - 1);
+        *dest++ = charset[index];
+    }
+    *dest = '\0';
+}
+
+
+internal 
+void WREN2C_encodeAndDump(ENGINE* engine, int argc, char* args[]) {
```

Comment:
It would be nice if this method was shared between DOME and the `utils/embed.c` tool, to avoid the duplication.

Reply:
If I remove the engine dependency, then it can be used as a stand alone function.
I just put it there for maybe using the logger. But can be ditched with no problems 👍

Reply:
Additionally. If this command would be available using the DOME executable.
Should embed.c be removed? Since it just a single main function with no relations to other DOME internals.

Reply:
No, we need the embed.c available separately to bootstrap DOME's compilation. 

Engine core developers might not have DOME installed before they want to start working on DOME ;)